src/Plugin.php

```diff
@@ -54,7 +54,7 @@ public static function init() {
    * @implements wp_enqueue_scripts
    */
   public static function enqueueAssets() {
-    if (!is_product() || has_term(Product::CATEGORY_EXCLUDED, 'product_cat')) {
+    if (!is_product() || Product::isTermExcluded()) {
```

Comment:
Maybe it is possible to keep `has_term`.

It seems to be support an array check 
https://developer.wordpress.org/reference/functions/has_term/#user-contributed-notes
https://stackoverflow.com/questions/48113310/multiple-has-term-in-if-statement

Reply:
@geist-ahnen — on a second look you are right, what confused me was the default of `''` (https://github.com/WordPress/WordPress/blob/9a517ac7aedce776cd8875c86eeb622d6922132a/wp-includes/category-template.php#L1535), I will change it back.